river/feature_extraction/vectorize.py

```diff
@@ -259,6 +271,28 @@ class BagOfWords(base.Transformer, VectorizerMixin):
     def transform_one(self, x):
         return collections.Counter(self.process_text(x))
 
+    def transform_many(self, X: pd.Series):
+        """Transform pandas series of string."""
+        X_p = [list(self.process_text(x)) for x in X]
+
+        idx, i = {}, 0
+        for x_p in X_p:
+            for x in x_p:
+                if x not in idx:
+                    idx[x] = i
+                    i += 1
+
+        array = np.zeros((len(X_p), len(idx)), dtype=int)
+
+        for row, x in enumerate(X_p):
+            for col in x:
+                array[row][idx[col]] += 1
+
+        return pd.DataFrame(array, columns=idx.keys(), index=X.index)
+
+    def learn_many(self, X):
+        pass
```

Comment:
Should be `return self`. In fact there should be a unit test that checks that `learn_one` and `learn_many` always return `self` (I think I did it for `learn_one`)